src/style.css

```diff
@@ -0,0 +1,31 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #000;
+}
+
+.box {
+  border: 1px solid #e75551;
+}
+
+.blue {
+  width: 350px;
+  background-color: #308dc0;
+  padding: 10px;
+  margin: 20px;
+  margin-bottom: 40px;
+}
+
+.orange {
+  background-color: #ffa500;
+  padding: 10px;
+}
+
+.content {
+  padding: 10px;
```

Comment:
You duplicate this style. 
You have one common class `box`, write this style to it.